pkg/apis/steward/v1alpha1/constants.go

```diff
@@ -43,7 +43,17 @@ const (
 	// faces an intermittent error during running phase.
 	EventReasonRunningFailed = "RunningFailed"
 
-	// EventReasonLoadPipelineRunsConfigFailed is the reason for a event occuring when the
+	// EventReasonLoadPipelineRunsConfigFailed is the reason for an event occuring when the
 	// loading of the pipeline runs configuration fails.
 	EventReasonLoadPipelineRunsConfigFailed = "LoadPipelineRunsConfigFailed"
+
+	// EventReasonSkipOnMaintenanceMode is the reason for an event occuring when a pipeline
+	// run is not started due to maintenance mode
+	EventReasonSkipOnMaintenanceMode = "SkipOnMaintenanceMode"
```

Comment:
"Skip" might be misleading, as pipeline runs are not skipped but just paused. What about just using "MaintenanceMode" as reason?

Reply:
What about PausedOnMaintenanceMode or DelayedOnMaintenanceMode?

Reply:
Took "MaintenanceMode" , see a547641e89ebab34bad0eaf87be23e83fb270c70